src/pages/Proposal/index.tsx

```diff
@@ -53,6 +57,19 @@ export const ProposalUi: React.FC<ProposalUiProps> = ({
     <div css={styles.root}>
       <ProposalSummary css={styles.summary} proposal={proposal} />
 
+      {!voteProposalFeatureEnabled && (
+        <div className="mb-8 w-full" data-testid={TEST_IDS.votingDisabledWarning}>
+          <NoticeInfo
```

Comment:
You can pass `className` and `data-testid` to `NoticeInfo` directly:
```suggestion
          <NoticeInfo className="mb-8 w-full" data-testid={TEST_IDS.votingDisabledWarning}
```